### PR TITLE
Add @NotNull 

### PR DESCRIPTION
**Pull request by Conclure**
Hi so I stumbled across some minor woopsies in the main plugin class. 

**Change 1**
As you know it's essential to know if a Lists element has  the possibilities of being null. Therefore I added @NotNull to the generic type parameter passed. 

**Change 2**
A static init block may not seem useful. But it can make code more readable as it often tears down one liners, resulting in two lines (declaration and assignment). This may be good because the code becomes less wide although longer in terms of height.

**Change 3**
I added the keyword this on an instance method invocation. This is firmly because of consistency since you're using it excessively in all other possible places. 

PS, don't take it seriously I'm just obsessed of becoming a contributor of this divine repository. Please merge 🙏

### CODE REVIEW

File: VeinMiner-Bukkit/src/main/java/wtf/choco/veinminer/VeinMiner.java [hunk 1]
```diff
@@ -334,7 +339,7 @@ private void registerAntiCheatHookIfEnabled(@NotNull PluginManager manager, @Not
         }
 
         AntiCheatHook hook = hookSupplier.get();
-        if (!registerAntiCheatHook(hook)) {
+        if (!this.registerAntiCheatHook(hook)) {
```
Comment (2008Choco) [inline VeinMiner-Bukkit/src/main/java/wtf/choco/veinminer/VeinMiner.java:342]: I generally avoid using `this.` if it's not the beginning of a statement.

File: VeinMiner-Bukkit/src/main/java/wtf/choco/veinminer/VeinMiner.java [hunk 2]
```diff
@@ -57,15 +57,20 @@
...
-    public static final Gson GSON = new Gson();
+    public static final Gson GSON;
 
-    public static final Pattern BLOCK_DATA_PATTERN = Pattern.compile("(?:[\\w:]+)(?:\\[(.+=.+)+\\])*");
+    public static final Pattern BLOCK_DATA_PATTERN;
+
+    static {
+      GSON = new Gson();
+      BLOCK_DATA_PATTERN = Pattern.compile("(?:[\\w:]+)(?:\\[(.+=.+)+\\])*");
+    }
```
Comment (2008Choco) [inline VeinMiner-Bukkit/src/main/java/wtf/choco/veinminer/VeinMiner.java:67]: I prefer the look of in-line initialization over static blocks.

**Review by 2008Choco — CHANGES_REQUESTED** [review]
The missing @NotNull annotation is fine, though I'd rather leave the rest as is.

**Comment by Conclure** [comment]
Changes made + new one liner when declaring including subprojects for the rootproject.

**Pull request merged**